### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by lukas-w**
This adds @tresf's issue templates for bug reports and feature requests as proposed in https://github.com/LMMS/lmms/pull/4196#issuecomment-372880674. When this is merged, clicking on `New Issue` will result in something similar to this example:

image

Issues created using these templates will automatically be assigned the `bug` or `enhancement` label respectively.

Supersedes  #4196.

### CODE REVIEW

**Comment by tresf** [comment]
One thing I recommend is making a checklist similar to how Homebrew does it.

https://github.com/Homebrew/homebrew-core/issues/new?template=Reproducible-Bug-Report.md

I don't necessarily think we need to be as harsh, but the general idea will make sure that we hold our bug submitters to a standard, specifically one that requires them to:
1. Perform basic troubleshooting and investigation  for bugs
2. Perform mockups for features
3. Forces them to search before posting

**Comment by claell** [comment]
I agree that at least the hint to search for duplicates should be added, don't know what to think of the "conversation" checklist style, as every issue will have unneeded checks, if I am correct.

Another probably helpful thing for bug reports might be the system details and LMMS version like:

```md
## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* LMMS version used:
* Operating System and version:
```

> Perform mockups for features

Do all feature requests require mockups?

**Comment by lukas-w** [comment]
I added a note about searching the issue tracker first. I don't believe using a checklist is beneficial unless we add more requirements. Also, it will show these little "task list" status indicators in the issue list:

image

I also added a link to Discord to guide users seeking help with using LMMS. You can try out the changes in my fork at https://github.com/lukas-w/lmms/issues/new/choose:

image

I'll merge this weekend if there are no objections. I think it's better than nothing, the main effect is hopefully making users think more about what they're posting before they do. We can always touch it up later if it's not sufficient.

**Comment by tresf** [comment]
`/offtopic` but @Reflexe can we add something like a "nobot" tag to prevent unnecessary builds on PRs that don't need them?

**Comment by Reflexe** [comment]
The bot does almost nothing and does not trigger any builds. I think that a
bot that actively looks for tags just to cancel the builds is overkill.
Especially when we have a pretty good caching system for compilation.
Maybe it is possible to configure CIs to ignore such PRs. I don't recall of
reading about it in any documentation.

**Comment by tresf** [comment]
> The bot does almost nothing and does not trigger any builds. I think that a bot that actively looks for tags just to cancel the builds is overkill.

I forgot..., let me rephrase... 

`/offtopic` but @Reflexe can we add something like a "nobot" tag to prevent **posting** builds on PRs that don't need them?

**Comment by lukas-w** [comment]
CircleCI and Travis recognize a `[skip ci]` tag which I guess I should have used for this PR. Not sure how the bot would have handled it if I did.

**Comment by tresf** [comment]
> CircleCI and Travis recognize a [skip ci] tag which I guess I should have used for this PR. Not sure how the bot would have handled it if I did.

Well, that solves it at the root of the problem.  Awesome.

**Comment by Reflexe** [comment]
> CircleCI and Travis recognize a `[skip ci]` tag which I guess I should have used for this PR. Not sure how the bot would have handled it if I did.

If the bot does not get notification of a successful build, it won't create a comment. It should not be that hard to implement. 

According to [this](https://circleci.com/docs/2.0/skip-build/) page, 
> Note: This feature is not supported for fork PRs. 

Next time open an issue on the lmms/bot repo instead, or else... 🗡  :)

**Comment by tresf** [comment]
> Next time open an issue on the lmms/bot repo instead, or else... 🗡 :)

Well, @lukas-w wouldn't have replied with his `skip ci` fix, so win/win. 💃

**Comment by lukas-w** [comment]
> image

Sorry, my comment was worded badly. You need `[skip ci]` in the commit message, not as a label of the PR.

`</offtopic>`

**Comment by tresf** [comment]
> Sorry, my comment was worded badly. You need [skip ci] in the commit message, not as a label of the PR.

Oh hell.  Ok @Reflexe wins.  I'll report there.
